compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td

```diff
@@ -499,6 +499,28 @@ def IREEGPU_TargetAttr : AttrDef<IREEGPU_Dialect, "Target"> {
     bool supportsTF32InputMMAOps() const;
     // Returns true if this target supports TensorCore synchronized MMA ops.
     bool supportsSyncMMAOps() const;
+    // Returns the SKU of the target GPU if available.
+    std::optional<StringRef> getSKU() const {
+        llvm::StringRef arch = getArch();
+
+        if (arch == "gfx942") {
```

Comment:
Invert this condition and use an early return: https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code